go/control/api/api.go

```diff
@@ -188,6 +193,37 @@ type RuntimeStatus struct {
 	Provisioner string `json:"provisioner,omitempty"`
 }
 
+// BundleStatus is the per-runtime bundle status overview.
+type BundleStatus struct {
+	// Name is the optional human readable runtime name.
+	Name string `json:"name,omitempty"`
+
+	// ID is the runtime identifier.
+	ID common.Namespace `json:"id"`
+
+	// Components contains statuses of the runtime components.
+	Components []ComponentStatus `json:"components,omitempty"`
+}
+
+// ComponentStatus is the component status overview.
+type ComponentStatus struct {
+	// Kind is the component kind.
+	Kind component.Kind `json:"kind"`
+
+	// Name is the name of the component.
+	Name string `json:"name,omitempty"`
+
+	// Version is the component version.
+	Version version.Version `json:"version,omitempty"`
+
+	// Detached specifies whether the component was in a detached bundled.
+	Detached bool `json:"detached,omitempty"`
+
+	// Disabled specifies whether the component is disabled by default
+	// and needs to be explicitly enabled via node configuration to be used.
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
This field is under section `bundle`, so it should tell you if the component is disabled in the bundle. If this setting is overridden in the runtime configuration, this should belong to `runtime` section.

The idea is to display available components, so that you can prepare configuration.

> If this is true, is the component currently disabled or was it just disabled by default and overridden in configuration?

I thought it is obvious from `needs to be explicitly enabled via node configuration to be used` that it is disabled by default.

Reply:
Ok.